src/main/java/io/vertx/core/eventbus/impl/EventBusImpl.java

```diff
@@ -488,6 +488,15 @@ public boolean send() {
     public Object sentBody() {
       return message.sentBody;
     }
+
+    @Override
+    public void forward(String address) {
+      message.setAddress(address);
+      if(message.send)
+        handlerRegistration.setRepliedAddress(address);
```

Comment:
this should not be changed, it is actually only used for providing more info for the metrics and by the time the value is changed, the registration will already have been done

Reply:
ok i removed registration modification